src/fides/api/ctl/migrations/versions/54102bba36de_add_attentive_connector.py

```diff
@@ -0,0 +1,74 @@
+"""add_attentive_connector
+
+Revision ID: 54102bba36de
+Revises: 68a518a3c050
+Create Date: 2023-03-09 22:08:17.480643
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "54102bba36de"
+down_revision = "68a518a3c050"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add to ConnectionType enum
+    op.execute("alter type connectiontype rename to connectiontype_old")
+    op.execute(
+        "create type connectiontype as enum('postgres', 'mongodb', 'mysql', 'https', "
+        "'snowflake', 'redshift', 'mssql', 'mariadb', 'bigquery', 'saas', 'manual', "
+        "'email', 'manual_webhook', 'timescale', 'fides', 'sovrn', 'attentive')"
+    )
+    op.execute(
+        (
+            "alter table connectionconfig alter column connection_type type connectiontype using "
+            "connection_type::text::connectiontype"
+        )
+    )
+    op.execute("drop type connectiontype_old")
+
+    # Rename field to something more generic
+    op.alter_column(
+        "privacyrequest",
+        "awaiting_consent_email_send_at",
+        new_column_name="awaiting_email_send_at",
+    )
+
+    # Rename awaiting_consent_email_send enum value to awaiting_email_send
+    op.execute(
+        "ALTER TYPE privacyrequeststatus RENAME VALUE 'awaiting_consent_email_send' TO 'awaiting_email_send'"
+    )
+
+
+def downgrade():
+    # Remove attentive from the connectiontype enum
+    op.execute("delete from connectionconfig where connectiontype in ('attentive')")
```

Comment:
typo here   which will keep downrev from running
op.execute("delete from connectionconfig where connection_type in ('attentive')")

Reply:
Fixed!